scripts/import/laptops/selfreport2csv

```diff
@@ -173,11 +176,13 @@ def load_data(
             print("redcap.export_records lacks a date_begin param, skipping")
         data = _load_data()
     
-    # Grabbing all reports
-    data['youth_report_2_timestamp'] = pd.to_datetime(data['youth_report_2_timestamp'], format='%Y-%m-%d %H:%M:%S', errors='coerce')
-    days_since_start = (datetime.now() - data['youth_report_2_timestamp']).astype('timedelta64[D]')
-    selected_idx = (data['youth_report_2_complete'] == '2') | (days_since_start > 30)
-    return data.loc[selected_idx]
+    # Grabbing necessary reports by default; return all if parser argument indicates
+    if (all == False):
+        data['youth_report_2_timestamp'] = pd.to_datetime(data['youth_report_2_timestamp'], format='%Y-%m-%d %H:%M:%S', errors='coerce')
```

Comment:
I think the last thing was that we can't use `youth_report_2_timestamp` because for incomplete surveys which we're trying not to pick up until they've been incomplete for a while, that field is always `[incomplete]` and therefore parses out as `NaN`. Instead, we should use `covid19_date`.

Reply:
(Which happens to be just YMD)

Reply:
Just saw this in the morning -- noted. Will fix by today!

Reply:
Attempting to print `data[covid19_date]` seems to output that the column doesn't exist, and searching for the column within the .csv output from REDCap also doesn't seem to indicate that there is a `covid19_date`. Should I be looking at another form?